test/suites/storage.sh

```diff
@@ -40,6 +40,20 @@ test_storage() {
       lxc storage volume show "$btrfs_storage_pool" "$btrfs_storage_volume"
       lxc storage volume set "$btrfs_storage_pool" "$btrfs_storage_volume" size 256MB
       lxc storage volume delete "$btrfs_storage_pool" "$btrfs_storage_volume"
+
+      # Test generation of unique UUID.
+      lxc init testimage uuid1 -s "lxdtest-$(basename "${LXD_DIR}")-pool-btrfs"
+      POOL="lxdtest-$(basename "${LXD_DIR}")-pool-btrfs"
+      lxc copy uuid1 uuid2
+      lxc start uuid1
+      lxc start uuid2
+      lxc stop --force uuid1
+      lxc stop --force uuid2
+      [ "$(blkid -s UUID -o value /dev/"${POOL}"/containers_uuid1)" != "$(blkid -s UUID -o value /dev/"${POOL}"/containers_uuid2)" ]
```

Comment:
I'm puzzled as to what is happening in these tests. Locally this works fine and for both btrfs and xfs new uuids are generated. Using `lsblk` will always show the same uuid which clearly is not the case. Using `blkid` will correctly show differing uuids in *some* cases and sometimes none at all. My only explanation is that somehow udev is not fast enough under heavy load. But I'm not sure.

Reply:
What does udev have to do with this? isn't blkid just reading the partition header?

Reply:
It uses `libuuid` I think. Also, `libblkid` uses a cache.

Reply:
There might be a trick: `blkid --cache-file=/dev/null`

Reply:
`blkid` returns empty in a bunch of cases meaning it doesn't list the uuid.